pkgs/applications/editors/bonzomatic/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, makeWrapper, fetchFromGitHub, cmake, alsaLib, mesa_glu, libXcursor, libXinerama, libXrandr, xorgserver }:
+
+let
+  name = "bonzomatic";
+in stdenv.mkDerivation {
+  inherit name;
```

Comment:
Please add a version in this package (the version is used by nix-env to upgarde). Something such as:
```
name = "${pname}-${version}"
pname = "bonzomatic";
version = "2018-03-29";
```